src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.Cos.cs

```diff
@@ -130,22 +130,22 @@ public static Vector128<float> Invoke(Vector128<float> x)
                 }
 
                 Vector128<float> almHuge = Vector128.Create(AlmHuge);
-                Vector128<float> dn = MultiplyAddEstimateOperator<float>.Invoke(uxMasked + Vector128.Create(float.Pi / 2), Vector128.Create(1 / float.Pi), almHuge);
+                Vector128<float> dn = FusedMultiplyAddOperator<float>.Invoke(uxMasked + Vector128.Create(float.Pi / 2), Vector128.Create(1 / float.Pi), almHuge);
```

Comment:
As noted in response to the main discussion thread...

AMD's does not require FMA and would not generate it on downlevel hardware.

We likewise should not require it because it will lead to poor performance on any hardware that pre-dates 2013, as well as for R2R code and the default NAOT experience.

Reply:
What do you suggest? Are you saying we should only use FMA for hardware that provides vectorized support for it?

Reply:
Yes, which is identical to how AOCL libm behaves when running on downlevel hardware.

For an individual `(x * y) + z`, FMA can only produce a max 1 ULP difference (that is the raw bitwise value is 1 bit higher or lower).

For the entire set of 7 multiply-adds, we should still be well within the 16 digit precision range for fused vs non fused.